pkg/rpmdb_testcase.go

```diff
@@ -2,387 +2,387 @@ package rpmdb
-		{0, "libXau", "1.0.6", "4.el6", "x86_64", "libXau-1.0.6-4.el6.src.rpm", 40009, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "upstart", "0.6.5", "17.el6", "x86_64", "upstart-0.6.5-17.el6.src.rpm", 567729, "GPLv2 and LGPLv2+", "CentOS", "", nil, nil, nil},
-		{0, "cloog-ppl", "0.15.7", "1.2.el6", "x86_64", "cloog-0.15.7-1.2.el6.src.rpm", 185386, "GPLv2+", "CentOS", "", nil, nil, nil},
-		{0, "gdb", "7.2", "92.el6", "x86_64", "gdb-7.2-92.el6.src.rpm", 5545845, "GPLv3+ and GPLv3+ with exceptions and GPLv2+ and GPLv2+ with exceptions and GPL+ and LGPLv2+ and GFDL and BSD and Public Domain", "CentOS", "", nil, nil, nil},
-		{0, "libthai", "0.1.12", "3.el6", "x86_64", "libthai-0.1.12-3.el6.src.rpm", 694712, "LGPLv2+", "CentOS", "", nil, nil, nil},
-		{0, "libedit", "2.11", "4.20080712cvs.1.el6", "x86_64", "libedit-2.11-4.20080712cvs.1.el6.src.rpm", 185648, "BSD", "CentOS", "", nil, nil, nil},
-		{0, "glibc-headers", "2.12", "1.212.el6_10.3", "x86_64", "glibc-2.12-1.212.el6_10.3.src.rpm", 2146171, "LGPLv2+ and LGPLv2+ with exceptions and GPLv2+", "CentOS", "", nil, nil, nil},
-		{0, "gcc", "4.4.7", "23.el6", "x86_64", "gcc-4.4.7-23.el6.src.rpm", 19499265, "GPLv3+ and GPLv3+ with exceptions and GPLv2+ with exceptions", "CentOS", "", nil, nil, nil},
-		{0, "libX11-common", "1.6.4", "3.el6", "noarch", "libX11-1.6.4-3.el6.src.rpm", 1341368, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "libXrender", "0.9.10", "1.el6", "x86_64", "libXrender-0.9.10-1.el6.src.rpm", 40515, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "cairo", "1.8.8", "6.el6_6", "x86_64", "cairo-1.8.8-6.el6_6.src.rpm", 797873, "LGPLv2 or MPLv1.1", "CentOS", "", nil, nil, nil},
-		{0, "libXrandr", "1.5.1", "1.el6", "x86_64", "libXrandr-1.5.1-1.el6.src.rpm", 43834, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "gdk-pixbuf2", "2.24.1", "6.el6_7", "x86_64", "gdk-pixbuf2-2.24.1-6.el6_7.src.rpm", 2637837, "LGPLv2+ and (LGPLv2+ or MPLv1.1) and Public Domain", "CentOS", "", nil, nil, nil},
-		{0, "libXdamage", "1.1.3", "4.el6", "x86_64", "libXdamage-1.1.3-4.el6.src.rpm", 24896, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "libXinerama", "1.1.3", "2.1.el6", "x86_64", "libXinerama-1.1.3-2.1.el6.src.rpm", 12039, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "pango", "1.28.1", "11.el6", "x86_64", "pango-1.28.1-11.el6.src.rpm", 1041857, "LGPLv2+", "CentOS", "", nil, nil, nil},
-		{0, "mailcap", "2.1.31", "2.el6", "noarch", "mailcap-2.1.31-2.el6.src.rpm", 52877, "Public Domain and MIT", "CentOS", "", nil, nil, nil},
-		{0, "perl-XML-Parser", "2.36", "7.el6", "x86_64", "perl-XML-Parser-2.36-7.el6.src.rpm", 652307, "GPL+ or Artistic", "CentOS", "", nil, nil, nil},
-		{0, "policycoreutils", "2.0.83", "30.1.el6_8", "x86_64", "policycoreutils-2.0.83-30.1.el6_8.src.rpm", 3596110, "GPLv2+", "CentOS", "", nil, nil, nil},
+		{0, "policycoreutils", "2.0.83", "30.1.el6_8", "x86_64", "policycoreutils-2.0.83-30.1.el6_8.src.rpm", 3596110, "GPLv2+", "CentOS", "SELinux policy core utilities", 1647005398, "", nil, nil, nil},
```

Comment:
Do you know why the order is updated?

Reply:
Maybe because that I regenerated the package database.

Reply:
Hmm. Why did you regenerate the package database? I saw many changes and it makes it hard to confirm the change is good.

Reply:
I did not see the test package database before, and then I saw them when I finished the patch.I'll generate a new patch later.